examples/common.yaml

```diff
@@ -87,6 +87,8 @@ openstack::neutron::password: 'whi-rtuz'
 openstack::neutron::shared_secret: 'by-sa-bo'
 
 ######## Ceilometer
+openstack::ceilometer::address::management: '192.168.11.4'
```

Comment:
For the example this should be the same as the controller management address, '172.16.33.4'.